features/tests_by_category_metrics.feature

```diff
@@ -0,0 +1,290 @@
+       "categories": [
+         {
+           "name": "C1",
+           "testsCount": 2
+         },
+         {
+           "name": "C2",
+           "testsCount": 1
+         }
+       ]
+      }
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve the number of tests by categories filtered by users
+    When hsolo sends a GET request to /api/metrics/testsByCategories?organizationId={@idOf: Rebel Alliance}&userIds[]={@idOf: lskywalker}
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      {
+        "noCategoryTestsCount": 1,
+        "categories": [
+          {
+            "name": "C2",
+            "testsCount": 1
+          },
+          {
+            "name": "C3",
+            "testsCount": 1
+          }
+        ]
+      }
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve the number of tests by categories filtered by projects and users
+    When hsolo sends a GET request to /api/metrics/testsByCategories?organizationId={@idOf: Rebel Alliance}&projectIds[]={@idOf: Y-Wing}&userIds[]={@idOf: lskywalker}
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      {
+        "noCategoryTestsCount": 0,
+        "categories": [
+          {
+            "name": "C3",
+            "testsCount": 1
+          }
+        ]
+      }
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve the number of tests by categories filtered by project version
+    When hsolo sends a GET request to /api/metrics/testsByCategories?organizationId={@idOf: Rebel Alliance}&projectVersionId[]={@idOf: 1.0.0}
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      {
+        "noCategoryTestsCount": 2,
+        "categories": [
+          {
+            "name": "C1",
+            "testsCount": 2
+          }
+        ]
+      }
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: An organization member should be able to retrieve the number of tests by categories filtered by project version and users
+    When hsolo sends a GET request to /api/metrics/testsByCategories?organizationId={@idOf: Rebel Alliance}&userIds[]={@idOf: lskywalker}&projectVersionId[]={@idOf: 1.0.0}
+    Then the response code should be 200
+    And the response body should be the following JSON:
+      """
+      {
+        "noCategoryTestsCount": 1,
+        "categories": [
+          {
+            "name": "C1",
+            "testsCount": 1
+          }
+        ]
+      }
+      """
+    And nothing should have been added or deleted
+
+
+
+  Scenario: A user should be able to retrieve the number of tests by categories from public organization
```

Comment:
"A member from another organization should be able to retrieve the number of tests by categories from a public organization" would be clearer.